pkg/controller/flink/job_manager_controller.go

```diff
@@ -92,6 +93,8 @@ func (j *JobManagerController) CreateIfNotExist(ctx context.Context, application
 	newlyCreated := false
 
 	jobManagerDeployment := FetchJobMangerDeploymentCreateObj(application, hash)
+	podSelector := jobManagerDeployment.Spec.Selector.MatchLabels[PodDeploymentSelector]
```

Comment:
The pod-deployment-selector on the JM is different from the TM deployment. I'm guessing this doesn't really matter, right? Do we require this label on the JM?

Reply:
Yeah, it doesn't matter. We just need _something_ to tie the pods to the deployment (and for the jobmanagers, to the services).